packages/components/src/templates/quickfact.stories.ts

```diff
@@ -0,0 +1,240 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Quickfact'
+};
+
+export const Default = {
+  render: () => html`
```

Comment:
I refined the quickfact template.
Could you update code accordingly, pls?

The grouping should not wrap ... should stay in 1 row ... 

We can fill it with real content later.

Reply:
Updated